lib/QuickBaseClient.rb

```diff
@@ -2328,8 +2323,8 @@ def findDBByname( dbname )
       @dbname = dbname
       xmlRequestData = toXML( :dbname, @dbname )
 
-      sendRequest( :findDBByname, xmlRequestData )
-      @dbid = getResponseValue( :dbid )
+      sendRequest( :GrantedDBs, xmlRequestData )
+      @dbid = getResponsePathValue( 'databases/dbinfo/dbid[1]' )

```

Comment:
@garethlatwork  Thanks for merging pull request.
In case of different user token dbid for USH  content manger is not found as first element (databases/dbinfo/dbid[1]) .. I have hard coded to [1] which is wrong.

Will generalise this and raise a pull request soon. Please consider.

Also once correct this, can you please  version and publish as gem ?